new_app/server.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * Created by omelvin on 5/21/15.
+ */
+
+var fs = require('fs');
+var util = require('util');
+var http = require('http');
+var path = require('path');
+var express = require('express');
+var passport = require('passport');
+var cookieParser = require('cookie-parser');
+var bodyParser = require('body-parser');
+var session = require('express-session');
+var mongoose = require('mongoose');
+
+var devMode = true;
+
+var dbConfig = require('./dbConfig');
+var mongoConnection = mongoose.createConnection(dbConfig.mongoURI, dbConfig.mongoConfig);
+if(devMode) {
+    mongoose.set('debug, true');
+}
+// Logging connection:
+mongoConnection.on('error', console.error.bind(console, 'DB connection error.')).once('open', console.log.bind(console, 'DB Connection established.'));
+
+var studentSchema = new mongoose.Schema({
+    firstName : String,
+    lastName  : String,
+    email     : String,
+    studentID : String,
+    className : String
+});
+
+var Student = mongoConnection.model('Student', studentSchema);
+
+var app = express();
+console.log(__dirname);
+//app.use(express.static(path.join(__dirname, 'public')));
+app.use(express.static(__dirname));
+app.use(cookieParser());
+app.use(bodyParser.urlencoded({ extended: true }));
+
+var registerHTML = fs.readFileSync('register.html').toString();
+app.get('/:className/register/:studentID', function(req, res) {
+    res.writeHead(200, {
+        'Content-Type': 'text/html'
+    });
+    res.end(registerHTML);
```

Comment:
with https://github.com/reactjs/express-react-views

this route could just be a one liner
res.render('register');